pkg/e2e/operators/cloudingress/secondary_router.go

```diff
@@ -69,16 +71,16 @@ var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
 				time.Sleep(time.Duration(120) * time.Second)
 			}
 
-			ingressControllerName := strings.Split(secondaryIngress.DNSName, ".")[0]
+			ingressControllerName := strings.Split(secondaryIngress.DNSName, ".")[1]
```

Comment:
To prevent a potential "index out of range" panic occurring here, it will be safer to `Expect()` that the length of the split array is > 1 first.

Reply:
Done